rust/azure_iot_operations_mqtt/src/session/dispatcher.rs

```diff
@@ -0,0 +1,205 @@
+use std::{
```

Comment:
add the copyright header in and (probably?) the doc comment describing what's here